userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -95,6 +104,28 @@ Fixed Term:: A type of <<Phase,phase>> that specifies an end date and during whi
 
 == I
 
+Invoice:: An invoice is a bill provided to a customer for charges that are payable on the customer's account. A single invoice consolidates charges for all subscriptions held by that customer for a specified time period. Invoices may be set to be paid automatically, or may be paid manually by the customer. An invoice consists of one or more invoice items.
+
+Invoice Item:: Represents a single item charged on an invoice. Given an active subscription, one could see multiple items for that subscription on a single invoice including recurring items, usage items, fixed price items, etc. There can also be items for different subscriptions on the same invoice, as well as items that are unrelated to subscriptions such as adjustments and taxes. 
+
+
+Invoice Item Type:: Specifies the type of invoice item. Can be one of `EXTERNAL_CHARGE`, `FIXED`, `RECURRING`, `REPAIR_ADJ`, `CBA_ADJ`, `ITEM_ADJ`, `USAGE`, `TAX`, `PARENT_SUMMARY`. Refer to the
```

Comment:
Is it intended to have two 'columns' `::` on all these?

Reply:
Yes, I think, I just copied @FlowWriter's style 😀  I tried removing the double colon (`::`)  and found that the document is not displayed properly.

Reply:
Right - this is the Asciidoctor syntax for glossary items. https://docs.asciidoctor.org/asciidoc/latest/sections/glossary/

Reply:
👍